v1.1/admin_ui_cluster_overview.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Cluster Overview 
+toc: false
+feedback: false
+---
+
+CockroachDB’s Admin UI enables you to monitor metrics such as SQL Queries, Service Latency, Replicas per Node, and Capacity of your cluster.
```

Comment:
Let's use the expresssion "The CockroachDB Admin UI" throughout instead of just "CockroachDB Admin UI" or "CockroachDB's Admin UI".

Reply:
Also, I think we can improve the intros to all dashboard pages by combining the text before and after the table of contents. I'd suggest a formulation like:

> The **Cluster Overview** dashboard in the Admin UI lets you monitor crucial metrics about SQL queries, SQL service latency, replicas, and cluster capacity. This dashboard is displayed by default as soon as you access the Admin UI.